.env.example

```diff
@@ -21,30 +11,32 @@ POSTGRES_PASSWORD=password
 POSTGRES_URL="postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@localhost:5432/db?schema=public"
 
 ## MongoDB
+# MONGO_USER=mongodb
```

Comment:
this is kinda a private thing, it's a free instance tho so ig it dun rlly matter